docs/json_rpc_api.md

```diff
@@ -1059,29 +1060,30 @@ func main() {
 Which results in:
 
 ```bash
-Block count: 276978
+2018/01/13 16:48:33 Block count: 203528
```

Comment:
I would prefer that the timestamp be left off.  I'm away that the log package adds one automatically, but it is going to constantly be out of date anyways, and it clutters up the example output in my opinion.

Reply:
OK, makes sense.